src/Diagnostics.DataProviders/ConfigurationFactory.cs

```diff
@@ -174,18 +175,76 @@ private void LoadConfigurationValues(object dataProviderConfiguration)
                         continue;
                     }
 
-                    var value = GetValue(prefix, attribute.Name);
+                    dynamic value;
+                    string valueType;
+                  
+                    if (property.Name == "OverridableExceptions")
```

Comment:
`OverridableExceptions` is really specific to Kusto, instead of constructing the overridableExceptionsList here, perhaps we can construct that in the `KustoDataProviderConfiguration` class?